sources/github/webhook-examples/issue-closed.json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
Right now the webhook examples need to have the format that the lambda expects. ie 

`{
    "payload": {
        "body": <your webhook payload here>
    }
}`

Reply:
example https://github.com/segmentio/functions-library/blob/master/sources/paypal/webhook-examples/billingPlanActivated.json